.taskcluster.yml

```diff
@@ -6,8 +6,8 @@ policy:
 tasks:
     - $let:
           taskgraph:
-              branch: taskgraph
-              revision: 0c5a68749f9a7672a7e56604b69a7bd41b036614
+              branch: taskgraph-try
+              revision: 3a2f4bee361ebdd3785c6b2f5cba2c2eef0d1e36
```

Comment:
We should definitely avoid using try in production when the taskgraph changes land.

Reply:
Yes. This is like this for testing before the taskgraph changes were ready/landed.